src/lib/jsonParse/jsonRequest.cpp

```diff
@@ -239,7 +239,7 @@ std::string jsonTreat
     std::string details = std::string("JSON parse error: ") + res;
     std::string answer;
     
-    alarmMgr.badInput(clientIp, details);
+    alarmMgr.badInput(clientIp, details, "");
```

Comment:
std::string details = std::string("JSON parse error");
      std::string answer;
 
      alarmMgr.badInput(clientIp, details, res);

Reply:
Ok

Reply:
Fixed in 10cdfce2c